keps/sig-node/127-user-namespaces/README.md

```diff
@@ -725,7 +787,7 @@ Pods will have to be re-created to use the feature.
 
 We will add.
 
-We will test for when the field pod.spec.HostUsers is set to true, false
+We will test for when the field pod.spec.hostUsers is set to true, false
```

Comment:
See also the comment from the KEP teplate:
```
Additionally, for features that are introducing a new API field, unit tests that
are exercising the `switch` of feature gate itself (what happens if I disable a
feature gate after having objects written with the new field) are also critical.
You can take a look at one potential example of such test in:
https://github.com/kubernetes/kubernetes/pull/97058/files#diff-7826f7adbc1996a05ab52e3f5f02429e94b68ce6bce0dc534d1be636154fded3R246-R282
```

that is relevant here and extend this answer.


Also - can you please add enablement/disablement tests eplicitly as beta graduation criteria?

Reply:
Oh, great call. Added that to this section and to the beta criteria!